app/jobs/audit_search_event_job.rb

```diff
@@ -3,6 +3,7 @@ class AuditSearchEventJob < SpreadsheetWriterJob
   queue_as :audit_search_event
 
   def perform(data)
+    return if WORKSHEET_POSITION.present? # Temporarily turn off this auditing until we decide what to do with it
```

Comment:
I can see why you've done this, but is there a risk that if we remove that variable the code will kick into life and then start erroring? It's a pretty low risk, so probably fine

Reply:
Yea Rubocop complained if I just put `return`. I hope the comment adds enough context to stop that from happening

Reply:
Yeah, it should be fine, and even if it isn't, all we'll get are errors